### DISCUSSION TITLE
Collection Expressions: Do not require [ ] when there is only one item in the expression and it is a collection

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#8441] [Language Ideas]
It would be really nice to simply be able to type:
```
property = .. collection,
```
instead of:
```
property = [.. collection],
```

**Comment by jnm2** [comment]
Not sure I agree. I'd start by thinking about spreading tuples first:
```cs
M(.. tuple1, .. tuple2); // Same as M(tuple1.A, tuple1.B, tuple2.A, tuple1.B);
```

Compared to:
```cs
M((.. tuple 1, .. tuple 2)); // Same as M((tuple1.A, tuple1.B, tuple2.A, tuple1.B));`
```

If womples existed, the same thing would hold. These two syntaxes would mean different things:
```cs
M(.. tuple);
M((.. tuple,));
```

I have the same intuition that `.. collection` and `[.. collection]` mean different things.

**Reply by jnm2** [reply]
I might even expect that `M(.. inlineArray4);` means `M(inlineArray[0], inlineArray[1], ...);`

**Reply by HaloFour** [reply]
Not to mention `..operand` is already legal syntax for a range.  My understanding was that it was specifically the collection expression syntax which allowed for it to be interpreted as a spread instead of as a range.

**Reply by jnm2** [reply]
Ah, that's even more important of a point. The syntactic model needs to decide which it is, based on syntax alone without knowing the type of anything or what names are in scope. So yes, `property = .. collection` will forever mean a range, I believe.
